src/main/java/uk/gov/companieshouse/web/pps/controller/pps/BankTransferPenaltyReferenceController.java

```diff
@@ -37,7 +43,8 @@ protected String getTemplateName() {
 
     @GetMapping
     public String getPenaltyReference(Model model) {
-        model.addAttribute(PPS_AVAILABLE_PENALTY_REF_ATTR, getAvailablePenaltyReferenceDisplay());
+        model.addAttribute(PPS_AVAILABLE_PENALTY_REF_ATTR,
+                penaltyConfigurationProperties.getAllowedRefStartsWith());
```

Comment:
Declare / instantiate this first before adding it here as same call used multiple times inside method.

```suggestion
                String allowedRefStartsWith = penaltyConfigurationProperties.getAllowedRefStartsWith());
```

Reply:
I think it's fine to leave this in the `getPenaltyReference` / `postPenaltyReference` methods as it is not called multiple times inside the same method.

Reply:
Yeah ignore me, going a bit mad there, separate methods, all good.